pack.go

```diff
@@ -10,7 +10,7 @@ import (
 	"github.com/buildpacks/pack/internal/style"
 )
 
-func extractPackagedBuildpacks(ctx context.Context, pkgImageRef string, fetcher ImageFetcher, publish, noPull bool) (mainBP dist.Buildpack, depBPs []dist.Buildpack, err error) {
+func extractPackagedBuildpacksFromImage(ctx context.Context, pkgImageRef string, fetcher ImageFetcher, publish, noPull bool) (mainBP dist.Buildpack, depBPs []dist.Buildpack, err error) {
```

Comment:
[not blocking] function name is long, IMHO and the parameters tell you its from an image so that's redudant.

Reply:
Fair point! I was going to make a similar method to extractPackagedBuildpacksFromFile, which was why I renamed it, but I don't think I'll do that in the end